ci-operator/step-registry/baremetal/lab/agent/install/baremetal-lab-agent-install-commands.sh

```diff
@@ -134,6 +135,103 @@ platform:
     ingressVIPs:
     - ${INGRESS_VIP}
 "
+else
+  yq --inplace eval-all 'select(fileIndex == 0) * select(fileIndex == 1)' "$SHARED_DIR/install-config.yaml" - <<< "
```

Comment:
Can we change this behavior through another step running before this one? 

Ideally, we can place the BMC hosts files within the SHARED_DIR as a file ending with `_patch_install_config.yaml` and be like:

```
platform:
  baremetal:
    hosts:
    - my host object 1
    - my host object 2
```

Then, without nesting other if statements, at line 237:

```
for f in $SHARED_DIR/*_patch_install_config.yaml;
do
  yq --inplace eval-all 'select(fileIndex == 0) * select(fileIndex == 1)' "$SHARED_DIR/install-config.yaml" $f
done

This pattern is already implemented in other places of the baremetal/ steps and in general in this repo, and will avoid so many changes and huge increase in size for central steps like this.